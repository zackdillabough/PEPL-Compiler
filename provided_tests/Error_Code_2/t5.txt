POLY A(R) = R + 2 R + 1;
POLY F(Q, W, Y) = Q^121 + W - R^2 - R;
POLY B(R) = R + 2 R + 1;
POLY C(W, C, D, S, J, R, S, F, A, T) = T + 1;
POLY D(X, x) = x + X + 
	y + z + 1;
POLY E = y + 
	x + x^2 -
	z - 1;
START
A(3);
INPUT X;
B(X);
INPUT Y;
INPUT Z;
F(X, Y, X);
INPUT X;
INPUT Y;
INPUT Z;
C(1, 2, 3, 4, 5, 6, 7, X, Y, X);
9 2 4 1 67 4
