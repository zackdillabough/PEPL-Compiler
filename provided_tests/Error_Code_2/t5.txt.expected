Error Code 2: 2 2 6 6 7 9 